site/docs/releases.md

```diff
@@ -71,6 +71,90 @@ To add a dependency on Iceberg in Maven, add the following to your `pom.xml`:
   ...
 </dependencies>
 ```
+
+### 1.6.0 release
+Apache Iceberg 1.6.0 was released on July 23, 2024.
+
+The 1.6.0 release contains fixes and dependency updates.
+
+* API
+    - Fix default FileIO#newInputFile ManifestFile, DataFile and DeleteFile implementation to pass lengths ([\#9953](https://github.com/apache/iceberg/pull/9953))
```

Comment:
This fix is already released in 1.5.1, no need to mention for 1.6.0
https://github.com/apache/iceberg/pull/10114

Reply:
It appears in the change log anyway. That's why I put there for consistency. I can remove it but I think it could be helpful for users.

Reply:
It was fixed in the main branch and then back ported to 1.5.x release. So, it appears in the change log. 

No need to mention it as it is not specific to this release.